workflow/rules/common.smk

```diff
@@ -459,16 +421,26 @@ def get_reference_name(reference_file):
 
 if not VPIPE_BENCH:
     reference_file = config["input"]["reference"]
-    if not os.path.isfile(reference_file):
+    if not reference_file:
```

Comment:
Do we want a `False` comparison or a comparison to `None`?

Reply:
(Note: this code is executed _after_ schema validation, so it's not possible to store a null by default as it would break the validator. JSON schema's defaults MUST also validate.
Whereas `""` is also a defaults that clearly states missing, also evaluates to `False` in Python,  but succesfully validate against a string schema)

Reply:
That makes sense!